src/pkg/utils/io.go

```diff
@@ -207,7 +219,17 @@ func CreatePathAndCopy(source string, destination string) error {
 		return err
 	}
 
-	return copy.Copy(source, destination)
+	// Copy all the source data into the destination location
+	if err := copy.Copy(source, destination); err != nil {
+		return err
+	}
+
+	// If the path doesn't exist yet then this is an empty file and we should create it
+	if InvalidPath(destination) {
+		return CreateFile(destination)
+	}
```

Comment:
```suggestion
	return CreateFile(destination)
```
We don't need the double check